spring-integration-core/src/main/java/org/springframework/integration/metadata/MetadataStore.java

```diff
@@ -51,11 +55,22 @@
 
 	/**
 	 * Remove a value for the given key from this MetadataStore.
+	 *
```

Comment:
We prefer do not add blank lines in the method Javadocs: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting